src/makeCalculator.js

```diff
@@ -1,10 +1,48 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
+
+    subtract(num) {
+      this.result -= num;
+
+      return this;
+    },
+
+    multiply(num) {
+      this.result *= num;
+
+      return this;
+    },
+
+    divide(num) {
+      if (num === 0) {
+        throw new Error('Error: Division by zero');
+      }
+      this.result /= num;
+
+      return this;
+    },
```

Comment:
[CODE STYLE] - Use arithmetic operators with assignment operator `=` if you want to add something to existing value. Although you've done this correctly in the `add`, `subtract`, and `multiply` methods, the `divide` method is missing this pattern. It should be `this.result /= num;` instead of the current implementation to follow the code style guide.